packages/types/src/sights.ts

```diff
@@ -140,6 +140,16 @@ export interface SightGuideline {
   nl: string;
 }
 
+/**
+ * The different orientations of a part selection.
+ */
+export enum PartSelectionOrientation {
+  FRONT_LEFT = 'front-left',
+  FRONT_RIGHT = 'front-right',
+  REAR_LEFT = 'rear-left',
+  REAR_RIGHT = 'rear-right',
```

Comment:
Add TSDoc for these fields

Reply:
can i add like 
`wireframe from front left`

Reply:
*Wireframe of the vehicle when looking at its front left side.*